recipe/build.sh

```diff
@@ -31,7 +34,7 @@ for i in `ls`; do
         fi
     else
         # Put all other files in targetsDir
-        mkdir -p ${PREFIX}/${targetsDir}/${PKG_NAME}
-        cp -rv $i ${PREFIX}/${targetsDir}/${PKG_NAME}
+        mkdir -p ${PREFIX}/${targetsDir}/${component_name}
+        cp -rv $i ${PREFIX}/${targetsDir}/${component_name}
```

Comment:
We were relying on the `PKG_NAME` to add some extra bits here. However now they go in `libcublas-split` (given that is when this build script runs

Have tried to address this here by creating a new variant/variable to handle this install location